commit 6f9feb773a43c5abfa3455da2e324180e789285b
Author: fanquake <fanquake@gmail.com>
Date:   Tue Sep 15 21:44:31 2020 +0800

    Fix PowerPC build of libpng

    See https://bugreports.qt.io/browse/QTBUG-66388.

    Can be dropped when we are building qt 5.12.0 or later.

diff --git a/qtbase/src/3rdparty/libpng/libpng.pro b/qtbase/src/3rdparty/libpng/libpng.pro
index 577b61d8..a2f56669 100644
--- a/qtbase/src/3rdparty/libpng/libpng.pro
+++ b/qtbase/src/3rdparty/libpng/libpng.pro
@@ -10,7 +10,7 @@ MODULE_INCLUDEPATH = $$PWD
 
 load(qt_helper_lib)
 
-DEFINES += PNG_ARM_NEON_OPT=0
+DEFINES += PNG_ARM_NEON_OPT=0 PNG_POWERPC_VSX_OPT=0
 SOURCES += \
     png.c \
     pngerror.c \
